be/src/io/fs/local_file_system.h

```diff
@@ -34,6 +34,8 @@ class LocalFileSystem final : public FileSystem {
 public:
     ~LocalFileSystem() override;
 
+    static Path convert_to_abs_path(const Path& path);
```

Comment:
add comment for this public static function. how does it behave
we have to emphasize that what is the path and what is the abs path.
what if the path is invalid?